File: API-Server/Happy Habits App/Forms/ExercisesWorkoutStatistics.cs
using System.Text.Json.Serialization;

namespace Happy_Habits_App.Forms
{
    public class ExercisesWorkoutStatistics(double averageDuration, List<string> topExercises, int totalWorkouts, double averageExercisesPerWorkout)
    {
        [JsonPropertyName("averageDuration")]
        public double AverageDuration { get; set; } = averageDuration;
        [JsonPropertyName("topExercises")]
        public List<string> TopExercises { get; set; } = topExercises;
        [JsonPropertyName("totalWorkouts")]
        public int TotalWorkouts { get; set; } = totalWorkouts;
        [JsonPropertyName("averageExercisePerWorkout")]
        public double AverageExercisesPerWorkout { get; set; } = averageExercisesPerWorkout;
    }
}

File: Mobile-App/app/src/main/java/com/example/happyhabits/feature_application/feature_workout/data/model/ExercisesWorkoutStatistics.kt
package com.example.happyhabits.feature_application.feature_workout.data.model

import kotlinx.serialization.Serializable

@Serializable

data class ExercisesWorkoutStatistics (
    val averageDuration: Double,
    val topExercises: List<String>,
    val totalWorkouts: Int,
    val averageExercisePerWorkout: Double
)